src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -169,6 +169,20 @@ public function isLegacy($utilPrefix)
             || \in_array('legacy', $test::getGroups($class, $method), true);
     }
 
+    /**
+     * @return bool
+     */
+    public function isMuted()
+    {
+        if (!isset($this->trace[0]['class'])) {
+            return false;
+        }
+
+        return 0 === strpos($this->trace[0]['class'], 'PHPUnit\\') && \in_array($this->message, [
```

Comment:
Should I bother supporting `PHPUnit_` here or will it never be useful to do so?

Reply:
no need IMHO, dealing only with the namespaced version is fine to me